hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1564,6 +1564,14 @@ public enum OperationStatusCode {
   // Default 10 mins.
   public static final int DEFAULT_SLOW_LOG_SYS_TABLE_CHORE_DURATION = 10 * 60 * 1000;
 
+  public static final String SLOW_LOG_OPERATION_JSON_MAX_COLS =
+    "hbase.regionserver.slowlog.operation.json.max.cols";
+  public static final int SLOW_LOG_OPERATION_JSON_MAX_COLS_DEFAULT = 50;
+
+  public static final String SLOW_LOG_OPERATION_JSON_ENABLED =
+    "hbase.regionserver.slowlog.operation.json.enabled";
```

Comment:
Would be good to remove `regionserver` from the config names, since the slowlog is used for master RPC calls as well.